config/routes.rb

```diff
@@ -1,15 +1,13 @@
 Rails.application.routes.draw do
   root "users#index"
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-end
 
-# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-# Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-# Can be used by load balancers and uptime monitors to verify that the app is live.
-get "up" => "rails/health#show", as: :rails_health_check
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:new, :create]
+    end
+  end
```

Comment:
- [ ] There should be a `:create` Like route, just as you have the create comment route.